packages/builder-vite/code-generator-plugin.ts

```diff
@@ -39,7 +39,7 @@ export function codeGeneratorPlugin(options: ExtendedOptions): Plugin {
       // HMR to update the importFn.
       server.watcher.on('add', (path) => {
         // TODO maybe use the stories declaration in main
-        if (/\.stories\.([tj])sx?$/.test(path) || /\.(story|stories).mdx$/.test(path)) {
+        if (/\.stories\.(([tj]sx?)|(vue))$/.test(path) || /\.(story|stories).mdx$/.test(path)) {
```

Comment:
```suggestion
        if (/\.stories\.(?:[tj]sx?|vue)$/.test(path) || /\.(story|stories).mdx$/.test(path)) {
```

Reply:
@JReinhold, this makes me wonder about `.stories.svelte` files too.  This is a pattern we use to detect when we need to trigger HMR changes in stories files.  We have a similar set of regexes in 7.0 as well.

Reply:
huh, interesting.
I think the todo above:
> // TODO maybe use the stories declaration in main
is the right approach here, but in the mean time we could just have a regex that matches on `*.stories.*`? It would match on too much, but in this case I don't think that will be an issue?